### PR TITLE
Tweaks for command line builds

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -559,12 +559,12 @@
 			remoteGlobalIDString = 19FF46011B8B4CF400B43179;
 			remoteInfo = "CocoaLumberjack-iOS";
 		};
-		CEA8847D1C191A72008D871B /* PBXContainerItemProxy */ = {
+		CEA884CB1C1A25D4008D871B /* PBXContainerItemProxy */ = {
 			isa = PBXContainerItemProxy;
 			containerPortal = CE0021551C03DBB70079DCA4 /* Lumberjack.xcodeproj */;
 			proxyType = 1;
-			remoteGlobalIDString = 19FF46011B8B4CF400B43179;
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj]: The static target should depend on the static ```CocoaLumberjack``` target for command line builds.

**Comment by bhariharan** [comment]
Outstanding item is fixing header path for ```CocoaLumberjack``` static target.

**Comment by khawkins** [comment]
:+1:

**Comment by wmathurin** [comment]
:+1:

**Pull request merged**